ground/src/main/java/com/google/android/ground/ui/offlineareas/selector/OfflineAreaSelectorViewModel.kt

```diff
@@ -79,7 +81,16 @@ internal constructor(
   private val _navigate = MutableSharedFlow<UiState>(replay = 0)
   val navigate = _navigate.asSharedFlow()
 
+  private val _showPopupEvent = MutableSharedFlow<Boolean>()
+  val showPopupEvent = _showPopupEvent.asSharedFlow()
```

Comment:
Can we rename this to `networkUnavailableEvent`? Also, we aren't using the value of the emitted value. So the type can be changed to `Nil`.